scss/bootstrap/overrides/style/_ui-bootstrap.scss

```diff
@@ -26,10 +25,6 @@
                         outline: 0 none;
                     }
 
-                    &:hover {
-                        background: $crm-brand-primary-lighter;
```

Comment:
we don't need this anymore?

Reply:
This is not affecting component design. it's tested